integration/301_internet_edge_with_ebpf_test.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Test short lived connections from the Internet"
+
+if ! echo "$HOST1" | grep "us-central1-a"; then
```

Comment:
Why is this needed? Why wouldn't it work locally? 

Also, hardcoding the zone is a hack. What if the machines are created somewhere else?